app/scripts/routes.js

```diff
@@ -16,6 +16,10 @@ define([], function() {
                 templateUrl: 'views/students/index.html',
                 controller: 'StudentsIndexCtrl',
                 relativePath: '/students'
+            },
+            '/students/:docket/update': {
+                templateUrl: '/views/students/update.html',
+                controller: 'StudentsUpdateCtrl'
```

Comment:
This controller should have set the `relativePath: '/students'` property, as the other controllers from above. When doing this, you should move this controller to the 'controllers/students' folder, as the others.